src/OptionBar.js

```diff
@@ -1,9 +1,113 @@
-import React from "react";
+import React, { useCallback, useContext, useState } from "react";
+import IdeaForm from "./IdeaForm";
+import { ThemeContext } from "./ThemeContext";
+import { IdeaContext } from "./IdeaContext";
+import update from "immutability-helper";
+import { v4 as uuidv4 } from "uuid";
+import "./OptionBar.css";
+import { SketchPicker } from "react-color";
 
 const OptionBar = () => {
+  const ideaPackage = useContext(IdeaContext);
+  const selectedId = ideaPackage.selectedId;
+  const selectedIdea = ideaPackage.ideas[selectedId];
+
+  const themePackage = useContext(ThemeContext);
+  const [colorSelectorValue, setColorSelectorValue] = useState(0);
+  const [background, setBackground] = useState("#800080");
+  const [hiding, setHiding] = useState({ display: "none" });
```

Comment:
Having possible values of `hiding` as `{ display: "none" }` and `{ display: "flex" }` is cryptic. I would make the possible values `true` or `false`.